astropy/io/ascii/misc.py

```diff
@@ -0,0 +1,124 @@
+"""A Collection of useful miscellaneous functions.
+
+misc.py:
+  Collection of useful miscellaneous functions.
+
+:Copyright: None -- Hack away at your leisure!!
```

Comment:
Would you be willing to put this under the same 3-clause BSD license as the rest of astropy? That's a very permissive license that still allows almost everybody to do everything. It just makes managing copyright issues so much easier if all files in a project use the same license. (And all files in astropy should have a header line to to refer to license.rst in the root directory - unfortunately `io.ascii` still has some legacy code in it, but if you look into any other part of astropy you will see it.